charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -45,6 +45,9 @@ spec:
         command:
         - ./machine-controller
         - --control-kubeconfig=inClusterConfig
+        - --machine-creation-timeout=20m
```

Comment:
Should we probably also remove these flags from the shared-MCM pods below, if it's not effective anymore, to avoid the confusion?

Reply:
I have kept it currently as these flags still exist are still used in the in-tree context. Let's deprecate them once all providers are migrated to OOT and get rid of them permanently.